### ISSUE TITLE
Update Footer and About text in Dockerised Archetype content

### ISSUE THREAD

**Issue by pastokes** [kcl-ddh/digipal#41]
The official Docker image of Archetype still has references to 'the DigiPal Framework' in the footer and About page, both of which should be replaced with Archetype. Suggested text is:

For the footer:
> Archetype is developed by the...

For the About page:
> This website is based on Archetype (formerly known as the DigiPal Framework). This is generic...

**Comment by geoffroy-noel-ddh** [comment]
Good point, thanks Peter. I'm not sure when exactly a new release can be scheduled but I'll integrate that change.

**Comment by pastokes** [comment]
Thanks Geoffroy, and yes, understood that there's no clear timeline. It's far from being a priority, anyway!
